src/content/app/genome-browser/contexts/genome-browser-ids-context/useGenomeBrowserUrlValidator.ts

```diff
@@ -220,6 +222,35 @@ const checkFocusLocation = async (params: CheckFocusObjectParams) => {
   return { isMissingFocusObject };
 };
 
+const checkFocusVariant = async (params: CheckFocusObjectParams) => {
+  // NOTE: so far, only checking that focus variant id conforms to the following format
+  // <region_name>:<start_coordinate>:<variant_name>
+  const { parsedFocusObjectId } = params;
+  const { objectId } = parsedFocusObjectId;
+
+  const variantIdParts = objectId.split(':');
+
+  if (variantIdParts.length !== 3) {
+    // expect three parts in a focus variant id
+    return {
+      isMissingFocusObject: true
```

Comment:
Isn't this incorrect focus object rather than missing focus object?

Reply:
Technically, yes; but in the future we are going to send the variant id to the variation api, and it will tell us that the variant with the supplied id does not exist.  I'd rather keep the error as missing focus object.

Reply:
If there is a `:` and no variant ID this condition becomes false which I guess is not correct. It should still capture the missing variant id

Reply:
> If there is a `:` and no variant ID this condition becomes false which I guess is not correct. It should still capture the missing variant id `focus=variant:1:230710048:`

Sorry this one is marked as resolved and you may miss it.